README.md

```diff
@@ -502,8 +502,8 @@ Navigate to the `_data/lang` folder and copy the `template` folder. Paste it int
 Translate the content of the files. Do not touch anything labeled `url`, and in the roadmap.yml ONLY translate the `name:` content.
 
 ### 14.3 _i18n folder
-Navigate to the _i18n folder and duplicate the template.yml file. Rename the duplicate to the two letter language code of your language with a `.yml` at the end. Repeat this process with the `template` folder as well.
-**The template folder and yml file themselves should still be there. They should not be renamed. There should be a new folder and yml file in addition to the ones that were already there.**
+Navigate to the _i18n folder and duplicate the en.yml file. Rename the duplicate to the two letter language code of your language with a `.yml` at the end. Repeat this process with the `en` folder as well.
```

Comment:
duplicated en.yml should be updated:
- set all sections to untranslated (translated = "no")
- remove all english sentences so that Jekyll will automatically pick up default (English) one.

Maybe a template.yml make sense here.

Reply:
updated with your suggestions. And yes, i agree a template here would be useful, even just to avoid to change all those 'yes' to 'no' every time